python/run-tests

```diff
@@ -126,7 +126,20 @@ function run_streaming_tests() {
       KAFKA_ASSEMBLY_JAR="$f"
     done
 
-    export PYSPARK_SUBMIT_ARGS="--jars ${KAFKA_ASSEMBLY_JAR} pyspark-shell"
+    KINESIS_ASL_ASSEMBLY_DIR="$FWDIR"/extras/kinesis-asl-assembly
```

Comment:
Would it be possible to move this logic into `pyspark/streaming/test.py`'s main method instead?  I'm in the middle of a Python test runner refactoring and it would greatly simplify things if the test suites themselves enforced that their prerequisites were installed rather than pushing that responsibility into the test triggering script.